isample/README.md

```diff
@@ -0,0 +1,32 @@
+
+# /isample/
+
+**Name of the project:** [iSamples](https://isamplesorg.github.io/home/)
+
+**Description:** redirects for resources related to sample registration and description.  Includes vocabularies for high level physical sample categorization--material type, sampled feature, and specimen type.
+
+requests to https://w3id.org/isample will redirect to the isample home page
+
+
+directory structure-- there are folders for each resource type, subfolders for subtypes within that resource type. Folders for different versions are nested within the folder for the resource that is versioned. 
+
+We are starting with vocabularies.
+
+Example:
+-isample
+    -- vocabulary
+        --- material
+           ---- 0.9
```

Comment:
I assume this should look like some sort of tree, so you need to escape it so it's not interpreted as markdown:
````suggestion
```
-isample
    -- vocabulary
        --- material
           ---- 0.9
```
````